src/voronoi.rs

```diff
@@ -158,41 +167,41 @@ mod test {
         let poly = vec![
             Polygon::new(
                 LineString(coords(vec![
+                    (0.5, 0.5),
```

Comment:
Changes here are to manually normalize the ordering within geometry collection and winding order of these polygons to match the output of normalizing the voronoi polygons.